src/lib/uPortMgr.js

```diff
@@ -1,14 +1,20 @@
-import { verifyJWT } from "did-jwt";
+import { verifyJWT, decodeJWT } from "did-jwt";
+
+const muportOpts = { ipfsConf: { protocol: 'https', host: 'ipfs.3box.io'}}
 
 require("uport-did-resolver")();
 require("ethr-did-resolver")();
-require("muport-did-resolver")();
+require("muport-did-resolver")(muportOpts);
 
 class UportMgr {
   async verifyToken(token) {
     if (!token) throw new Error("no token");
     return verifyJWT(token);
```

Comment:
If this throws an error we can register the muport-did-resovler without the muportOpts arg and try again.